lib/committee/drivers/open_api_2.rb

```diff
@@ -133,6 +133,9 @@ def call
                 param_schema.items = param_data["items"]
               end
 
+              # if parameter is defined in header, uppercase name parameter to recognize CGI header.
+              param_data["name"].upcase! if param_data["in"] == "header"
```

Comment:
Can we just perform the `upcase` when doing the validation? I think having all the logic in one place is just going to make everything in a little more readable.